BusBoard.ConsoleApp/Program.cs

```diff
@@ -18,15 +18,13 @@ static void Main()
                 }
 
                 var busEntries = ApiHelper
-                    .ApiGet<BusEntry>("https://api.tfl.gov.uk/", $"StopPoint/{stopPoint}/Arrivals").ToList();
+                    .ApiGet<BusEntry>("https://api.tfl.gov.uk/", $"StopPoint/{stopPoint}/Arrivals");
 
-                busEntries = busEntries.OrderBy(busEntry => busEntry.expectedArrival).ToList();
-                busEntries = busEntries.GetRange(0, 5);
+                busEntries = busEntries.OrderBy(busEntry => busEntry.expectedArrival);
+                busEntries = busEntries.Take(5);
```

Comment:
It's preferrable to avoid reassigning to the same variable in a function as it makes the logic that clear. You can either combine your statements or create new variables. In this case I'd just combine them all:
```
var busEntries = ApiHelper
  .ApiGet<BusEntry>("https://api.tfl.gov.uk/", $"StopPoint/{stopPoint}/Arrivals")
  .OrderBy(busEntry => busEntry.expectedArrival)
  .Take(5);
```
🙂

Reply:
Oh that's a nice way of arranging combined statements. I'll definitely be using that in the future.